definitions/divorce/json/CaseEvent/CaseEvent-pet-amend-nonprod.json

```diff
@@ -22,18 +9,5 @@
         "PreConditionState(s)": "AosCompleted;AwaitingAmendCase",
         "PostConditionState": "AmendPetition",
         "SecurityClassification": "Public"
-    },
-    {
-      "LiveFrom": "29/06/2020",
-      "CaseTypeID": "DIVORCE",
-      "ID": "amendPetitionForRefusalSol",
-      "Name": "Submit amended application",
-      "Description": "Submit amended application",
-      "DisplayOrder": 3,
-      "PreConditionState(s)": "DNisRefused;Issued;Rejected;PendingRejection;AosAwaiting;AosStarted;AosSubmittedAwaitingAnswer;AwaitingDecreeNisi;AosOverdue;AwaitingReissue;AosCompleted;AosAwaitingSol;AosPreSubmit;AosDrafted;AwaitingService;AwaitingAmendCase",
-      "PostConditionState": "AmendPetition",
-      "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/solicitor-amend-petition-dn-rejection",
-      "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
-      "SecurityClassification": "Public"
```

Comment:
Moved to '-nonprod` file

Reply:
nonprod was meant for different scenarios (things that should never be deployed - like "testEventToMoveCaseToGivenState")